### PR TITLE
Add gopass.gopass version 1.15.5

### PR DESCRIPTION
**Pull request by slokhorst**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

See upstream issue https://github.com/gopasspw/gopass/issues/1949
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106108)

### CODE REVIEW

**Comment by slokhorst** [comment]
The Url validation error should be fixed now. Can the check be re-run?

/AzurePipelines run

File: manifests/g/gopass/gopass/1.15.5/gopass.gopass.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: gopass.gopass
+PackageVersion: 1.15.5
+MinimumOSVersion: 7.0.0.0
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: GnuPG.Gpg4win
```
Comment (stephengillie) [inline manifests/g/gopass/gopass/1.15.5/gopass.gopass.installer.yaml:8]:
```suggestion
```

Comment (stephengillie) [reply manifests/g/gopass/gopass/1.15.5/gopass.gopass.installer.yaml:8]: This package installed and launched normally without the dependency. Does having the dependency enable additional features or functionality within the package?

**Comment by stephengillie** [comment]
The `Kleopatra` application launched normally from the desktop shortcut. The `gopass` shortcut causes a window to pop up and disappear, whether clicked or run from PowerShell. Going to its target, `%appdatalocal%\gopass\gopass.exe`, and running that gives:
```

   __     _    _ _      _ _   ___   ___
 /'_ '\ /'_'\ ( '_'\  /'_' )/',__)/',__)
( (_) |( (_) )| (_) )( (_| |\__, \\__, \
'\__  |'\___/'| ,__/''\__,_)(____/(____/
( )_) |       | |
 \___/'       (_)

≡ƒîƒ Welcome to gopass!
ΓÜá No existing configuration found.
Γÿ¥ Please run 'gopass setup'

Error: not initialized
```
Is this the expected behavior?

**Comment by slokhorst** [comment]
Hi @stephengillie, thanks for your review!

> Is this the expected behavior?

Yes, that is correct.

You can then run `gopass.exe setup` to do the initial configuration. That step will only work if GnuPG.Gpg4win is installed, which is why I listed it as a dependency. (if it is not installed, the program will exit with the message "failed to init crypto backend: failed to detect binary: no gpg binary found")

File: manifests/g/gopass/gopass/1.15.5/gopass.gopass.installer.yaml (hunk 1 above)
Comment (slokhorst) [reply manifests/g/gopass/gopass/1.15.5/gopass.gopass.installer.yaml:8]: Yes, this dependency is needed for basic functionality. See https://github.com/microsoft/winget-pkgs/pull/106108#issuecomment-1542742626

**Comment by stephengillie** [comment]
I'm not sure why Wingetbot is removing the waiver.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**